cmd/worker/main.go

```diff
@@ -35,7 +36,27 @@ const (
 	localPkgPathFmt = "/local/%s"
 )
 
-func handleMessage(ctx context.Context, msg *pubsub.Message, packagesBucket *blob.Bucket, resultsBucket, fileWritesBucket, imageTag string) error {
+type notificationMessage struct {
```

Comment:
This type describes a message format that will probably be relied on externally, so I think maybe it should be in a separate file.

In fact, perhaps it can go into the `resultstore` package, which means that the Pkg type can remain unexported.

Reply:
Hmm, actually, maybe it's better to have a separate `notification` package, which uses its own internal `pkgIdentifier` type - at some point it would be nice to have a single type representing a package, but for now having separate unexported types is probably fine

Reply:
+1 to having a single type representing a package, but I'll create a new package for `notification` for now